packages/codemods/src/codeMods/mods/configMod/configMod.mod.ts

```diff
@@ -131,9 +131,9 @@ const configMod: CodeMod = {
   run: (file: SourceFile) => {
     if (!__configs) {
       __configs = getCodeModsFromJson();
-      if (__configs === undefined || __configs.length === 0) {
+      if (__configs === undefined) {
         return Err<ModResult, ModError>({
-          error: 'failed to get any mods from json. Perhaps the file is missing or malformed?',
+          error: 'Failed to get any mods from json. Perhaps the file is missing or malformed?',
```

Comment:
This had probably been meant not to fail the codemod pipeline. But it changed in #17932 accidentally